Sources/Swift/Persistence/SentryScopePersistentStore.swift

```diff
@@ -0,0 +1,222 @@
+@_implementationOnly import _SentryPrivate
+
+@objc
+enum SentryScopeField: UInt, CaseIterable {
+    case context
+    case user
+    
+    var name: String {
+        switch self {
+        case .context:
+            return "context"
+        case .user:
+            return "user"
+        }
+    }
+}
+
+@_spi(Private) @objc public protocol SentryFileManagerProtocol {
```

Comment:
`m`: Please move this protocol to a different file, so if we reuse it for other classes unrelated to the `SentryScopePersistentStore` it's not referencing this file (modular file structure)